modules/transport-netty4/build.gradle

```diff
@@ -42,13 +42,6 @@ dependencies {
     compile "io.netty:netty-transport:4.1.6.Final"
 }
 
-integTest {
-    includePackaged = true
-    cluster {
-        numNodes = 2
```

Comment:
there are no regular rest tests here? there is only the one that checks that the plugin is loaded?